test/offsetarray.jl

```diff
@@ -268,8 +268,7 @@ am = map(identity, a)
 # other functions
 v = OffsetArray(v0, (-3,))
 @test_approx_eq v v
-@test parent(v') == v0'
```

Comment:
There are two parents inVolved here now. The RowVector is the outermost wrapper. We could do `parent(parent(v')) == parent(v0')` but I wasnt sure what it was testing.